docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+############################################################
+A: Yes, e.g DALI was used to achieve high performance in NVIDIA’s MLPerf submission for UNet3D.
+You can read an interesting article about it `here <https://developer.nvidia.com/blog/accelerating-medical-image-processing-with-dali>`_.
+You can see the DALI pipeline that was used `in this example <https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/Segmentation/nnUNet/data_loading/dali_loader.py>`_.
+
+Q: Where can I find more details on using the image decoder and doing image processing?
+#######################################################################################
+A: You can always refer to `the relevant section of the DALI documentation <../examples>`_
+where you can find multiple examples of DALI used in different use-cases. For the start,
+you can also watch `our introductory talk on this GTC <https://www.nvidia.com/gtc/session-catalog/#/session/1636566824182001pODM>`_.
+
+Q: Does DALI utilize any special NVIDIA GPU functionalities?
+############################################################
+A: Yes, DALI uses `NVJPEG <https://developer.nvidia.com/blog/loading-data-fast-with-dali-and-new-jpeg-decoder-in-a100/>`_ -
+special HW unit offloading JPEG image decoding, `NVDEC <https://developer.nvidia.com/nvidia-video-codec-sdk>`_ -
+HW video decoder, `GPUDirect Storage <https://developer.nvidia.com/gpudirect-storage>`_ -
+the ability to load data directly to the GPU to avoid a slow round trip through CPU.
+
+Q: Can DALI operate without GPU?
+################################
+A: Yes, DALI can operate without GPU but it is not optimized for the CPU itself.
+The main goal of this functionality is to enable the development of the DALI pipeline on
+machines where GPU is not available (like laptops), with an ability to later deploy the DALI
+pipeline on a GPU-capable cluster.
+
+Q: Can I use DALI in the Triton server through a Python model?
+##############################################################
+A: You could do that if the Python used by the server has DALI installed but for
+the best performance, we encourage you to use the dedicated DALI backend. It skips
+the Python layer and optimizes the interaction between the Triton server and the DALI pipeline.
+
+Q: Can the Triton model config be auto-generated for a DALI pipeline?
+#####################################################################
+A: Not yet but we are actively working on that feature and we expect to provide
+model config auto-generation for the DALI backend soon.
+
+Q: How can we decide whether to use RAPIDS(cuDF) or DALI? What are the strengths/weaknesses of either that are not present in the other?
+########################################################################################################################################
+A: DALI is best suited for dense data such as images, video, audio, etc,
+while RAPIDS is better suited for data analytics and ML where data is tabular.
```

Comment:
How is nvtabular related to RAPIDS/cuDF?

Reply:
cuDF is one of libraries inside RAPIDS, nvtabular is build on top of it.